SoftLayer/CLI/virt/bandwidth.py

```diff
@@ -36,7 +36,14 @@ def cli(env, identifier, start_date, end_date, summary_period, quite_summary):
     """
     vsi = SoftLayer.VSManager(env.client)
     vsi_id = helpers.resolve_id(vsi.resolve_ids, identifier, 'VS')
-    data = vsi.get_bandwidth_data(vsi_id, start_date, end_date, None, summary_period)
+
+    # Summary period is broken for virtual guests, check VIRT-11733 for a resolution.
+    # For now, we are going to ignore summary_period and set it to the default the API imposes
+    if summary_period != 300:
+        click.secho("""The Summary Period option is currently set to the 300s as the backend API will throw an exception
+any other value. This should be resolved in the next version of the slcli.""", fg='yellow')
+    summary_period = 300
```

Comment:
always 300

Reply:
yes this is expected. If you send in ANY rollupSeconds value to SoftLayer_Metric_Tracking_Object::getBandwidthData (with a virtual guest object anyway) you get the error `SoftLayerAPIError(500): Metric Service call failed after attempting to connect to these hosts, in order:
web-agent-proxy-bds-metrics.app.com.okd.softlayer.local:80`. But if you send in no value, the data that comes back is in 5 minutes / 300 second increments.

Reply:
Once the API is fixed, I will revert this code, but I'm not sure how long that will take